Sources/OTelSpanConventions/Generated/SpanAttributes+http.swift

```diff
@@ -0,0 +1,268 @@
+                public var method: Self.Key<MethodEnum> { .init(name: OTelAttribute.http.request.method) }
+                
+                public struct MethodEnum: SpanAttributeConvertible {
+                    private let rawValue: String
+                    /// `CONNECT`: CONNECT method.
+                    public static let connect = Self.init(rawValue: "CONNECT")
+                    /// `DELETE`: DELETE method.
+                    public static let delete = Self.init(rawValue: "DELETE")
+                    /// `GET`: GET method.
+                    public static let get = Self.init(rawValue: "GET")
+                    /// `HEAD`: HEAD method.
+                    public static let head = Self.init(rawValue: "HEAD")
+                    /// `OPTIONS`: OPTIONS method.
+                    public static let options = Self.init(rawValue: "OPTIONS")
+                    /// `PATCH`: PATCH method.
+                    public static let patch = Self.init(rawValue: "PATCH")
+                    /// `POST`: POST method.
+                    public static let post = Self.init(rawValue: "POST")
+                    /// `PUT`: PUT method.
+                    public static let put = Self.init(rawValue: "PUT")
+                    /// `TRACE`: TRACE method.
+                    public static let trace = Self.init(rawValue: "TRACE")
+                    /// `_OTHER`: Any HTTP method that the instrumentation has no prior knowledge of.
+                    public static let other = Self.init(rawValue: "_OTHER")
+                    public func toSpanAttribute() -> Tracing.SpanAttribute {
+                        return .string(self.rawValue)
+                    }
+                }
+        
+                /// `http.request.method_original`: Original HTTP method sent by the client in the request line.
+                /// 
+                /// - Stability: stable
+                /// 
+                /// - Type: string
+                /// 
+                /// - Examples:
+                ///     - `GeT`
+                ///     - `ACL`
+                ///     - `foo`
+                public var methodOriginal: Self.Key<String> { .init(name: OTelAttribute.http.request.methodOriginal) }
+        
+                /// `http.request.resend_count`: The ordinal number of request resending attempt (for any reason, including redirects). 
+                /// 
+                /// - Stability: stable
+                /// 
+                /// - Type: int
+                /// 
+                /// The resend count SHOULD be updated each time an HTTP request gets resent by the client, regardless of what was the cause of the resending (e.g. redirection, authorization failure, 503 Server Unavailable, network issues, or any other). 
+                /// 
+                /// - Example: `3`
+                public var resendCount: Self.Key<Int> { .init(name: OTelAttribute.http.request.resendCount) }
+            }
+        
+        
```

Comment:
This might be the same fix that could remove the extra newlines from the `OTelAttribute+x` files.